pkg/operator/resources/namespaced/apiserver.go

```diff
@@ -0,0 +1,124 @@
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package namespaced
+
+import (
+	appsv1 "k8s.io/api/apps/v1"
+	corev1 "k8s.io/api/core/v1"
+	rbacv1beta1 "k8s.io/api/rbac/v1beta1"
+	"k8s.io/apimachinery/pkg/util/intstr"
+)
+
+const (
+	apiServerRessouceName     = "cdi-apiserver"
+	extensionAPIResourceName  = "cdi-extension-apiserver-authentication"
+	extensionAPIConfigMapName = "extension-apiserver-authentication"
+)
+
+func createAPIServerResources(args *FactoryArgs) []Resource {
+	return []Resource{
+		createAPIServerServiceAccount(),
+		createAPIServerRoleBinding(args.Namespace),
+		createAPIServerRole(),
+		createExtensionAPIServerRoleBinding(args.Namespace),
+		createExtensionAPIServerRole(),
+		createAPIServerService(),
+		createAPIServerDeployment(args.DockerRepo, args.APIServerImage, args.DockerTag, args.Verbosity, args.PullPolicy),
+	}
+}
+
+func createAPIServerServiceAccount() *corev1.ServiceAccount {
+	return createServiceAccount(apiServerRessouceName)
+}
+
+func createAPIServerRoleBinding(serviceAccountNamespace string) *rbacv1beta1.RoleBinding {
+	return createRoleBinding(apiServerRessouceName, apiServerRessouceName, apiServerRessouceName, serviceAccountNamespace)
+}
+
+func createAPIServerRole() *rbacv1beta1.Role {
+	role := createRole(apiServerRessouceName)
+	role.Rules = []rbacv1beta1.PolicyRule{
+		{
+			APIGroups: []string{
+				"",
+			},
+			Resources: []string{
+				"secrets",
+				"configmaps",
+			},
+			Verbs: []string{
+				"get",
+				"create",
+			},
+		},
+	}
+	return role
+}
+
+func createExtensionAPIServerRoleBinding(serviceAccountNamespace string) *rbacv1beta1.RoleBinding {
+	roleBinding := createRoleBinding(
+		extensionAPIResourceName,
+		extensionAPIResourceName,
+		apiServerRessouceName,
+		serviceAccountNamespace,
+	)
+	roleBinding.Namespace = "kube-system"
+	return roleBinding
+}
+
+func createExtensionAPIServerRole() *rbacv1beta1.Role {
+	role := createRole(extensionAPIResourceName)
+	role.Rules = []rbacv1beta1.PolicyRule{
+		{
+			APIGroups: []string{
+				"",
+			},
+			Resources: []string{
+				"configmaps",
+			},
+			Verbs: []string{
+				"get",
+			},
+			ResourceNames: []string{
+				extensionAPIConfigMapName,
+			},
+		},
+	}
+	return role
+}
+
+func createAPIServerService() *corev1.Service {
+	service := createService("cdi-api", "cdi.kubevirt.io", apiServerRessouceName)
+	service.Spec.Ports = []corev1.ServicePort{
+		{
+			Port: 443,
+			TargetPort: intstr.IntOrString{
+				Type:   intstr.Int,
+				IntVal: 8443,
+			},
+			Protocol: corev1.ProtocolTCP,
+		},
+	}
+	return service
+}
+
+func createAPIServerDeployment(repo, image, tag, verbosity, pullPolicy string) *appsv1.Deployment {
+	deployment := createDeployment(apiServerRessouceName, "cdi.kubevirt.io", apiServerRessouceName, apiServerRessouceName, 1)
```

Comment:
constant for cdi.kubevirt.io

Reply:
common.CDIComponentLabel is the constant name.